### DISCUSSION TITLE
Ideal Channel for newbies `first-timers` OR `good-first-issues`

### DISCUSSION THREAD

**Discussion by kaiwalyakoparkar** [EddieHubCommunity/support#1746] [Poll] [answered]
We encountered that there are 2 channels `first-timers` & `good-first-issues` in the discord server which are meant to provide good first issues to the newbies. But both the channels have mixed issues of `advance or help` level and `good-first-time-issues`. So, this discussion is to decide:

**Comment by verma-kunal** [comment]
- According to me, we can name one channel as `Green Squares Arena`, which would be definite and a place for all the new folks to find beginner-friendly issues & the discussions related to that (only)

**Comment by kaiwalyakoparkar** [comment, marked as answer]
1. Which channel should serve which need?
2.  If we have to rename the channel which is not for the beginner then what should be its name according to you?
